fluentlenium-core/src/main/java/org/fluentlenium/core/annotation/Page.java

```diff
@@ -6,6 +6,10 @@
 import java.lang.annotation.RetentionPolicy;
 import java.lang.annotation.Target;
 
+/**
+ * @deprecated in favor of {@link javax.inject.Inject}.
+ */
+@Deprecated
```

Comment:
Is it an option to make a Page annotation a reference to Inject just to not force everyone to change theirs annotations in existing frameworks? It seems like huge change to everyone and I'm thinking how to reduce effort on users end.